src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package mate.jdbc.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElse(new Car());
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        car.getDrivers().add(driver);
+        carDao.update(car);
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        car.getDrivers().remove(driver);
+        carDao.update(car);
```

Comment:
Maybe it is better to write something like that:
```suggestion
        List<Driver> drivers = car.getDrivers();
        drivers.add(driver)
        car.setDrivers(drivers);
        carDao.update(car);
```